src/Sentry/Sentry.csproj

```diff
@@ -7,7 +7,7 @@
     <RootNamespace>Sentry</RootNamespace>
     <Description>Official SDK for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
     <NoWarn Condition="$(TargetFramework) == 'netstandard2.0'">$(NoWarn);RS0017</NoWarn>
-    <DefineConstants>$(AdditionalConstants)</DefineConstants>
+    <DefineConstants>$(AdditionalConstants);TRACE</DefineConstants>
```

Comment:
Does the trace logger not work without `TRACE` constant?

Reply:
I wonder if we're adding overhead to the SDK by setting this. We're not explicitly adding anything that depends on `TRACE` constant being defined so I assume it wouldn't matter but it does make me wonder. Does it affect the size of the binary at all for example?

Reply:
Trace.Write/WriteLine doesn't work without the `TRACE` trace constant. Trace.Listeners are exposed with or without the TRACE constant, but in my opinion, it's better to use set the TRACE constant and using the build-in Trace.Write instead of reinventing the wheel.

The package sizes are:
1,076,098 bytes with the TRACE constant
1,076,158 bytes without the TRACE constant and without the Trace Logger file
1,077,102 bytes without it.
1,077,858 bytes without the trace constant with the addition of a loop to write to all listeners.
1,080,993 bytes adding the DEBUG constant (out of curiosity, this allows the debug diagnostic logger to work properly in release)